example/datasource/consul/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+
+	"github.com/alibaba/sentinel-golang/ext/datasource"
+	"github.com/alibaba/sentinel-golang/ext/datasource/consul"
+	"github.com/hashicorp/consul/api"
+)
+
+func main() {
+	ds := startConsulDs()
+
+	scanner := bufio.NewScanner(os.Stdin)
```

Comment:
Here we may make it a real demo (with `sentinel.Entry(xxx)`) so that users could observe the effect as soon as the rules have been modified.